docs/02-app/01-building-your-application/04-styling/03-css-in-js.mdx

```diff
@@ -9,19 +9,19 @@ description: Use CSS-in-JS libraries with Next.js
 >
 > We're working with the React team on upstream APIs to handle CSS and JavaScript assets with support for React Server Components and streaming architecture.
 
-The following libraries are supported in Client Components in the `app` directory:
+The following libraries are supported in Client Components in the `app` directory (alphbetical):
 
+- [`pandacss`](https://panda-css.com)
 - [`styled-jsx`](#styled-jsx)
 - [`styled-components`](#styled-components)
-- [`tamagui`](https://tamagui.dev/docs/guides/next-js#server-components)
 - [`style9`](https://github.com/johanholmerin/style9)
+- [`tamagui`](https://tamagui.dev/docs/guides/next-js#server-components)
 - [`vanilla-extract`](https://github.com/vercel/next.js/tree/canary/examples/with-vanilla-extract)
 
 The following are currently working on support:
 
 - [`emotion`](https://github.com/emotion-js/emotion/issues/2928)
 - [Material UI](https://github.com/mui/material-ui/issues/34905#issuecomment-1401306594)
-- [Chakra UI](https://github.com/chakra-ui/chakra-ui/issues/7180)
```

Comment:
Double checking... Was this removed on purpose?

Reply:
Yes, Panda is from the creators of Chakra. Maybe we need to make that more clear.

Reply:
Chakra is still actively working on supporting this as well. I don't think it should be removed